flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/operations/UseModulesOperation.java

```diff
@@ -37,4 +41,16 @@ public List<String> getModuleNames() {
     public String asSummaryString() {
         return String.format("USE MODULES: %s", moduleNames);
     }
+
```

Comment:
Not related to this PR, 
however since this class is touched.
Doesn't it make sense to init `moduleNames` as unmodified collection once and then return it rather than create a new unmodified list per getter call?

Reply:
Good point!